state/logs.go

```diff
@@ -564,11 +591,20 @@ func (t *logTailer) tailOplog() error {
 			}
 			rec, err := logDocToRecord(t.modelUUID, doc)
 			if err != nil {
-				return errors.Annotate(err, "deserialization failed (possible DB corruption)")
+				if deserialisationFailures == 0 {
+					logger.Warningf("deserialization failed (possible DB corruption), %v", err)
+				}
+				deserialisationFailures++
+				continue
+			} else {
+				if deserialisationFailures > 1 {
```

Comment:
Doesn't this mean that after the first error it'll keep logging after every received log entry?